.pkgmeta

```diff
@@ -20,6 +20,7 @@ externals:
   WeakAuras/Libs/LibDBIcon-1.0: https://repos.wowace.com/wow/libdbicon-1-0/trunk/LibDBIcon-1.0
   WeakAuras/Libs/LibGetFrame-1.0: https://github.com/mrbuds/LibGetFrame
   WeakAuras/Libs/Archivist: https://github.com/emptyrivers/Archivist
+  WeakAuras/Libs/LibSerialize: https://github.com/rossnichols/LibSerialize
```

Comment:
As mentioned elsewhere, you should ingest version 1.0.0 so I don't have to worry about breaking WeakAuras :)